hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/client/ScmClient.java

```diff
@@ -392,6 +392,15 @@ StartContainerBalancerResponseProto startContainerBalancer(
    */
   List<String> getScmRatisRoles() throws IOException;
 
+  /**
+   * Get the current SCM mode.
+   *
+   * @return `true` indicates that it is in RATIS mode,
+   * while `false` indicates that it is in STANDALONE mode.
+   * @throws IOException  an I/O exception of some sort has occurred.
+   */
+  boolean isSCMRatisEnable() throws IOException;
```

Comment:
isSCMRatisEnable -> isScmRatisEnable

Reply:
Thank you for reviewing the code! I will improve this part of the code.